packages/std/src/math/fraction.rs

```diff
@@ -10,6 +10,7 @@ pub trait Fraction<T>: Sized {
     /// Returns the multiplicative inverse `q/p` for fraction `p/q`.
     ///
     /// If `p` is zero, None is returned.
+    #[must_use = "this returns the result of the operation, without modifying the original"]
```

Comment:
Is this the right place here? Does this belong to the trait definition or the trait implementation?

Reply:
Thanks for double checking